src/storage/ws-browser.ts

```diff
@@ -138,7 +138,7 @@ export class WebSocketBrowserStorageProvider implements StorageProvider {
       const serviceId = this.services.get(url);
       if (serviceId) {
         this.deleteService(serviceId).catch((e) =>
-          this.logger.warn(`[WebSocketBrowserStorageProvider] Failed to delete service ${serviceId}: ${e}`),
+          this.logger.error(`[WebSocketBrowserStorageProvider] Failed to delete service ${serviceId}: ${e}`),
```

Comment:
Please restore the logging level and push `serviceId` and `error` into the context.